Ports/julius/package.sh

```diff
@@ -13,11 +13,31 @@ depends=(
 configopts=(
     "-DCMAKE_TOOLCHAIN_FILE=${SERENITY_BUILD_DIR}/CMakeToolchain.txt"
 )
+install_dir='/opt/julius'
+launcher_name='Julius'
+launcher_category='Games'
+launcher_command="${install_dir}/julius"
+icon_file='res/julius_32.png'
 
 configure() {
     run cmake "${configopts[@]}" .
 }
 
 install() {
-    run make "${installopts[@]}" install
+    run_nocd mkdir -p "${SERENITY_INSTALL_ROOT}${install_dir}/"
+    run cp -r julius "${SERENITY_INSTALL_ROOT}${install_dir}/"
+    # This is necessary to allow the user to create save files
+    run_nocd chmod a+w "${SERENITY_INSTALL_ROOT}${install_dir}/"
+    run_nocd mkdir -p "${SERENITY_INSTALL_ROOT}/usr/local/bin/"
+    run ln -sf "${install_dir}/julius" "${SERENITY_INSTALL_ROOT}/usr/local/bin/julius"
+}
+
+post_install() {
+    echo
+    echo 'Julius is installed!'
+    echo
+    echo 'Make sure your game files are present in the following directory:'
+    echo "    Inside SerenityOS: ${install_dir}/"
+    echo "    Outside SerenityOS: ${SERENITY_SOURCE_DIR}/Base${install_dir}/"
```

Comment:
I'm personally leaning towards installing files in `Root` rather than `Base`, since the former isn't tracked by Git.

Reply:
I've changed this to use `SERENITY_INSTALL_ROOT`  instead (with `realpath` to remove the extra forward slash)

There are a few other ports that do this as well. We should probably make sure that they are all consistent.